data-source/jsonnet/england-wales/individual/blocks/personal-details/confirm_dob.jsonnet

```diff
@@ -1,7 +1,7 @@
 local placeholders = import '../../../lib/placeholders.libsonnet';
 local rules = import 'rules.libsonnet';
 
-local question(title, yesLabel, noLabel, noValue) = {
+local question(title, yesLabel, noLabel, noValue, yesValue) = {
```

Comment:
The ordering of the variables is not consistent. It should be `yesLabel, yesValue, noLabel, noValue`.